scene/resources/gradient.cpp

```diff
@@ -69,7 +69,12 @@ void Gradient::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_interpolation_mode", "interpolation_mode"), &Gradient::set_interpolation_mode);
 	ClassDB::bind_method(D_METHOD("get_interpolation_mode"), &Gradient::get_interpolation_mode);
 
+	ClassDB::bind_method(D_METHOD("set_interpolation_color_space", "color_space"), &Gradient::set_color_space);
```

Comment:
The C++ method names should match the one of the bindings. (set/get_interpolation_color_space)
For consistency, I'd suggest also renaming the member variable color_space to interpolation_color_space.